books/free-programming-books-es.md

```diff
@@ -283,7 +283,7 @@
 
 #### Redis
 
-* [El pequeño libro Redis](https://raulexposito.com/the-little-redis-book-en-castellano.html) - Karl Seguin, `trl.:` Raúl Expósito (HTML, PDF, EPUB)
+* [El pequeño libro Redis](https://raulexposito.com/the-little-redis-book-en-castellano.html) - Karl Seguin, `trl.:` Raúl Expósito (HTML, [PDF](https://raulexposito.com/assets/pdf/redis.pdf), [EPUB](https://raulexposito.com/assets/epub/redis.epub))
```

Comment:
Not needed since there are a link the landing page.

Reply:
Do you mind takin a look at https://github.com/EbookFoundation/free-programming-books/pull/7637#discussion_r990340739? Not sure how to proceed but a proper answer could unblock both PRs. 
Thank you